app/lib/theme/widgets/ReadOnlyAdjustableCalculatedValueWidget.tsx

```diff
@@ -21,10 +22,11 @@ const ReadOnlyAdjustableCalculatedValueWidget: React.FC<WidgetProps> = (
           {
             <NumberFormat
               thousandSeparator
-              fixedDecimalScale={isMoney}
+              fixedDecimalScale={isMoney || isPercent}
               id={id}
               prefix={isMoney ? "$" : ""}
-              decimalScale={isMoney ? 2 : 10}
+              suffix={isPercent ? "%" : ""}
```

Comment:
we're missing that suffix + decimalScale change in the `<NumberFormat .../>` just below